preferences/Preference.js

```diff
@@ -1,19 +1,23 @@
 // See https://github.com/jenkinsci/js-storage
-const storage = require('@jenkins-cd/storage');
-const jenkinsNS = storage.jenkinsNamespace();
-const preferences = jenkinsNS.subspace('preferences');
+const storage = require('@jenkins-cd/storage/storage');
+const StorageNamespace = require('@jenkins-cd/storage/StorageNamespace');
```

Comment:
In fact, I don't think there's a need to use `StorageNamespace` directly anyway. I think you can just do the following...

```javascript
const storageNS = storage. localNamespace('thorsetens-ns');
```